src/main/java/mops/foren/domain/model/OwnRights.java

```diff
@@ -0,0 +1,16 @@
+package mops.foren.domain.model;
+
+import java.util.EnumSet;
+
+public class OwnRights {
```

Comment:
Why do you wrap this again? Can't you set this field in the permissionManager?

Also we agreed to call it "Permissions" and not "Rights"

Reply:
This Class got deleted in the latest commit, it also says outdated on the top of the file

Reply:
I see

Reply:
Approved.